docs/build/unitary-synthesis.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: Synthesizing unitary matrices
+description: On the implementation of arbirtary unitary matrices on qubits
+---
+
+A unitary operation describes a norm-preserving change to a quantum system.
+For $n$ qubits is described by a $2^n \times 2^n$ dimensional, complex matrix $U$ whose adjoint equals the inverse, that is $U^\dagger U = \mathbb{1}$.
+
+Synthesizing specific unitary operations into a set of quantum gates is a fundamental task used, for example, in the design and application of quantum algorithms or in compiling quantum circuits.
+
+While efficient synthesis is possible for certain classes of unitaries—like those composed of Clifford gates or having a tensor product structure—most unitaries do not fall into these categories.
+For general unitary matrices, synthesis is a complex task with computational costs that increase exponentially with the number of qubits.
+Therefore, if you know an efficient decomposition for the unitary you would like to implement, it will likely be better than a general synthesis.
+
+If no decomposition is available Qiskit provides you with the tools to find one. 
+However, note that this generally generates deep circuits that may be unsuitable to run on noisy quantum computers.
```

Comment:
I think this note could be put into a "Note" admonition box

Reply:
Do you have an example of that syntax? 🙂